lib/beaker/hypervisor/abs.rb

```diff
@@ -74,5 +80,78 @@ def provision
     def cleanup
       # nothing to do
     end
+
+    def provision_vms(hosts)
+
+      verbose = false
+      config = Conf.read_config # get the vmfloaty config file in home dir
+
+      # TODO: the options object provided by the floaty cli is required in get_service_config()
+      # we should make it optional or accept nil
+      cli = Object.new
+      def cli.service() "abs" end
```

Comment:
note this is the `--service abs` which means that hard-coded value would look for the equivalent service in the vmfloaty.yml file. eg it would need one called "abs" and use that one.
If we set it to nil, it will pick up the first one, would that be better for the use-case here?
Otherwise we could add a way to pass in that parameter.

Reply:
I think hard-coded works. Looking over our confluence migration to abs docs this matches up with that. I've never seen ABS referred to as anything else.

Reply:
I think this could easily trip up any of us who dev on abs as I doubt I am the only one who has multiple abs instances in my floaty config. Is it practical to make it configurable somehow?

Reply:
ok I'll look into this, maybe default to this but also accept ENV vars?

Reply:
Totally make sense - just some way to pick something else.